src/Tribe/Timezones.php

```diff
@@ -94,6 +94,36 @@ public static function wp_timezone_abbr( $date ) {
 	}
 
 	/**
+	 * Generates a timezone string from a UTC offset
+	 *
+	 * @param string $offset UTC offset
+	 *
+	 * @return string
+	 */
+	public static function generate_timezone_string_from_utc_offset( $offset ) {
+		if ( ! self::is_utc_offset( $offset ) ) {
+			return $offset;
+		}
+
+		// ensure we have the minutes on the offset
+		if ( ! strpos( $offset, ':' ) ) {
+			$offset .= ':00';
+		}
+
+		$offset = str_replace( 'UTC', '', $offset );
+
+		list( $hours, $minutes ) = explode( ':', $offset );
+		$seconds = $hours * 60 * 60 + $minutes * 60;
+
+		$timezone = timezone_name_from_abbr( '', $seconds, 1 );
```

Comment:
... thinking it may not matter for the week view issue at hand, but are there other places this method might be used from in future where it could introduce hard-to-find weirdness and so could it be worth adding a caveat in the docblock about this behaviour?

If you feel that's overkill, that's fine too :+1: